apps/pie-microsite/src/assets/styles/components/_tokensTable.scss

```diff
@@ -88,8 +89,15 @@ $tokensTable-border-width: 1px;
 
     @include f.media('>=wide') {
         padding: f.spacing(e) 0;
-        border-width: 0 0 $tokensTable-border-width 0;
         border-radius: 0;
+        border-width: 0 0 $tokensTable-border-width 0;
+
+        // Remove bottom border for final item in non-section token lists and the final item in section token lists
```

Comment:
This comment doesn't make a huge amount of sense (as someone not familiar with the code).

I'm guessing it's something to do with the subcategories listings, but I'd make this comment more verbose and as clear as possible.